app/src/main/java/com/rafaelmardom/movies_and_shows_api/domain/Repositories.kt

```diff
@@ -0,0 +1,6 @@
+package com.rafaelmardom.movies_and_shows_api.domain
+
+interface MovieRepository {
+    fun getMovies(): List<Movie>
```

Comment:
the name of method: getMoviees or getMovieById repeat 'movie'. You can use:
getAll() and getById or findById...

Reply:
Done!